.travis.yml

```diff
@@ -3,11 +3,11 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - 0.7
   - 1.0
+  - 1.1
 notifications:
   email: false
+before_script:
+  - julia -e 'using Pkg; Pkg.add(PackageSpec(name="MathOptInterface", rev="master"))'
```

Comment:
Add a TODO to remove this before merging?

Reply:
When MOI is released, I remove this to lines and push to trigger the new build. I'll leave this comment unresolved as a reminder to be sure